dev-ml/opam/Manifest

```diff
@@ -1,2 +1 @@
-DIST opam-1.3.1.tar.gz 669485 BLAKE2B e1aed54175d46f5dd9fd989e56c25657370356e26df6976c01dd089e381291fa04e25da295104f85d513010cd028276d80ebf7a4a77b38f3e089e52f4e64f43a SHA512 d075c9b2c9670d712f29bd2df34b8020da52041fa2bbef599cb54a81f63554507d78bebece45ecf668d3d1d4ed514e4ab88a1d86b64088d283bca7a8e75ae582
```

Comment:
We can drop this change as it’s been ported

Reply:
It's still an old ~arch ebuild while newer version is stable.

Reply:
Yes, but it got restored for a reason I can’t check right now, we can ask @tupone but I’m pretty sure it’s for the old ~arch only version of OCaml.

Reply:
Yeah that should be checked. Since CI is green it means at least there is no such dependency expressed in case old OCaml would be broken.